### PR TITLE
Update bambora client to use basepath

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

update config to force bambora service to take basepath

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [X] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [X] Yes
- [ ] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ services:
     ports:
       - "8080:8080"
     environment:
+      - BAMBORA_APIBASEPATH=${BAMBORA_APIBASEPATH:-http://localhost:8080}
```
Comment (alexjoybc) [inline docker-compose.yml:54]: not is use but yes

Comment (alexjoybc) [reply docker-compose.yml:54]: Can't you assign a default in the yaml

**Review by alexjoybc — CHANGES_REQUESTED** [review]
default the config in the yml of the api

File: docker-compose.yml (hunk 1 above)
Comment (TayGov) [reply docker-compose.yml]: 👍 done 🏎️

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**